File: Api_Recordis/src/main/java/com/recordis/api_recordis/model/Usuario.java
package com.recordis.api_recordis.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document(collection = "usuarios")
public class Usuario {

    @Id
    private String id;

    private String nombre;
    private int edad;
    private String avatar;
    private String color;
    private String genero;
    private String foto;
}

File: recordis/domain/entities/Usuario.js
// domain/entities/Usuario.js
export const crearEntidadUsuario = (raw) => ({
  id: raw.id ?? raw._id,
  nombre: raw.nombre,
  edad: raw.edad,
  avatar: raw.avatar ?? '👤',
  color: raw.color ?? '#3B82F6',
  genero: raw.genero ?? 'Hombre',
});
